src/components/welcome/index.tsx

```diff
@@ -1,57 +1,86 @@
   const router = useRouter()
+  const wallet = useWallet()
+  const connectWallet = useConnectWallet()
 
   return (
-    <>
-      <Typography variant="h1" mb={2}>
-        Welcome to the Safe.
-      </Typography>
-      <Typography variant="h4" mb={2}>
-        Safe is the most trusted platform to manage digital assets.
-        <br />
-        Here is how to get started:
-      </Typography>
-      <Paper sx={{ padding: 3, maxWidth: '800px' }}>
-        <Grid container gap={3}>
-          <Grid item md={6}>
-            <Typography variant="h2" mb={1}>
-              Create Safe
-            </Typography>
-            <Typography mb={4}>
-              Create a new Safe that is controlled by one or multiple owners. You will be required to pay a network fee
-              for creating your new Safe.
-            </Typography>
-            <Track {...CREATE_SAFE_EVENTS.CREATE_BUTTON}>
-              <Button variant="contained" onClick={() => router.push(AppRoutes.newSafe.create)}>
-                + Create new Safe
-              </Button>
-            </Track>
-            <Divider orientation="vertical" flexItem />
-          </Grid>
-          <Grid item md>
-            <Typography variant="h2" mb={1}>
-              Load Existing Safe
-            </Typography>
-            <Typography variant="body1" mb={4}>
-              Already have a Safe or want to access it from a different device? Easily load your Safe using your Safe
-              address.
-            </Typography>
-            <Track {...LOAD_SAFE_EVENTS.LOAD_BUTTON}>
-              <Button variant="outlined" onClick={() => router.push(AppRoutes.newSafe.load)}>
-                Add existing Safe
-              </Button>
-            </Track>
-          </Grid>
-        </Grid>
-      </Paper>
-    </>
+    <Grid container spacing={3}>
+      <Grid item xs={12} md={4} lg={3.5}>
+        <div className={css.sidebar}>
+          <SafeList closeDrawer={() => {}} />
+        </div>
+      </Grid>
+      <Grid item xs={12} md={8} lg={8.5}>
+        <div className={css.content}>
+          <Typography variant="h1" fontSize={[44, null, 52]} lineHeight={1} letterSpacing={-1.5} mb={1}>
+            Welcome to the Safe
+          </Typography>
+          <Typography mb={5}>
+            The most trusted decentralized custody protocol and collective asset management platform.
+          </Typography>
+          {wallet ? (
+            <Grid container spacing={3} sx={{ maxWidth: '800px' }}>
+              <Grid item xs={12} sm={6}>
+                <Paper sx={{ padding: 4, height: 1 }}>
+                  <SvgIcon component={NewSafeIcon} inheritViewBox sx={{ width: '42px', height: '42px' }} />
+                  <Typography variant="h3" fontWeight={700} mb={1} mt={3}>
+                    Create Safe
+                  </Typography>
+                  <Typography variant="body2" mb={3}>
+                    New Safe that is controlled by one or multiple owners.{' '}
```

Comment:
This copy definitely needs work, but as it stands it lacks an "A" at the beginning.